Justfile

```
@@ -184,22 +184,15 @@ _build-bib $target_image $tag $type $config: (_rootful_load_image target_image t
     #!/usr/bin/env bash
     set -euo pipefail
 
-    mkdir -p "output"
-
-    echo "Cleaning up previous build"
-    if [[ $type == iso ]]; then
-      sudo rm -rf "output/bootiso" || true
-    else
-      sudo rm -rf "output/${type}" || true
-    fi
-
     args="--type ${type} "
     args+="--use-librepo=True"
 
     if [[ $target_image == localhost/* ]]; then
       args+=" --local"
     fi
 
+    TEMPDIR=$(mktemp -d)
```

Consider setting the base directory for `mktemp` to be on the locally cloned repo or `/var/tmp`. Raw "`mktemp -d`" will create a directory in `/tmp` which can be usually mounted as a `tmpfs` with about 2 or so GBs of RAM and that cant support an entire ISO
```suggestion
    TEMPDIR=$(mktemp -d -p .)
```

---

I updated it to follow the conventions set previously as closely as I could.